public/nginx.conf

```
@@ -28,6 +28,17 @@ http {
     include       /etc/opt/rh/rh-nginx112/nginx/mime.types;
     default_type  application/octet-stream;
 
+	gzip on;
```

```suggestion
    gzip on;
```

LGTM if you can fix the difference in indentation w/ surrounding lines